reset_security_server.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+# Script should be run as sudo
```

Comment:
I do not think  'sudo' is normally necessary, the group memberships seem to be set up quite sanely. On my Ubuntu I run ``lxc`` commands without sudo, being member of ``lxd`` group.

Reply:
Removed that requirement